DCCpp/PacketRegister.cpp

```diff
@@ -468,6 +481,11 @@ void RegisterList::printPacket(int nReg, byte *b, int nBytes, int nRepeat) volat
   INTERFACE.print(">");
 } // RegisterList::printPacket()
 
+byte RegisterList::idlePacket[3]={0xFF,0x00,0};                           // always leave extra byte for checksum computation
+byte RegisterList::resetPacket[3]={0x00,0x00,0};    // fnd - what is this tagged on? Not Gregg's original code
```

Comment:
Not sure I understand the Q, but this packet should be referenced for usage on the PROG track APIs to signal the decoders to reset (it is literally three zero bytes as a packet, checksum is zero)

Reply:
These last two lines are in your fork of the code but didn't seem to be in Gregg's original.  Yet looking directly at his Github, they are!? Why is this appended at the end of this module? I'm used to definitions going at the top of code somewhere. I am really confused as to when I was doing a diff, these lines weren't there and I added them :(

Reply:
I didn't add them, I'm not 100% certain why they are shown as a diff other than the comments at the end of lines.

https://github.com/DccPlusPlus/BaseStation/blob/master/DCCpp_Uno/PacketRegister.cpp#L473-L476